tutorials/btp-app-events-app-setup-s4hc/btp-app-events-app-setup-s4hc.md

```diff
@@ -0,0 +1,378 @@
+primary_tag: software-product-function>sap-cloud-application-programming-model
+---
+
+
+## Prerequisites
+ - [Add the Consumption of an External Service to Your CAP Application](btp-app-ext-service-add-consumption)
+ - [Consume the External Service in the UI of Your Application](btp-app-ext-service-consume-ui)
+ - [Register Your SAP S/4HANA Cloud System](btp-app-ext-service-s4hc-register)
+ - [Use Your SAP S/4HANA Cloud Service for Your Deployed CAP Application](btp-app-ext-service-s4hc-use)
+ - When starting with the result from the [`ext-service-s4hc-use`](https://github.com/SAP-samples/cloud-cap-risk-management/tree/ext-service-s4hc-use) branch, refer to [Create a Directory for Development](btp-app-create-directory) to see a recommended approach for organizing directories and details how to copy the content of the branch.
+ - On SAP BTP side:
+    - You have an [enterprise](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/171511cc425c4e079d0684936486eee6.html) global account in SAP BTP.
+    - You must be an administrator of the SAP BTP global account where you want to register your SAP S/4HANA Cloud system.
+    - You need to [Prepare for SAP BTP Development](btp-app-prepare-btp) if you start with a result from an example branch.
+    - Your SAP BTP subaccount has quota for the services `SAP Launchpad service` and `SAP HTML5 Applications Repository service` as described in [Prepare for SAP BTP Development](btp-app-prepare-btp).
+    - You have to [Use an existing SAP HANA Cloud service instance](https://developers.sap.com/tutorials/btp-app-hana-cloud-setup.html#42a0e8d7-8593-48f1-9a0e-67ef7ee4df18) or [Set up a new SAP HANA Cloud service instance](https://developers.sap.com/tutorials/btp-app-hana-cloud-setup.html#3b20e31c-e9eb-44f7-98ed-ceabfd9e586e) for the deployment. After the deployment, you need to perform step 14–17, starting with step [Subscribe to SAP Launchpad Service](https://developers.sap.com/de/tutorials/btp-app-launchpad-service.html#57352c79-1a09-4054-a77d-626fac957404) from the tutorial [Add the SAP Launchpad service](btp-app-launchpad-service).
+ - On SAP S/4HANA Cloud side:
+    - You have a dedicated SAP S/4HANA Cloud tenant.
+    - You must be an administrator of the SAP S/4HANA Cloud system.
+    - You need to connect this system to your SAP BTP global account, if you'd like to build extension applications for your SAP S/4HANA Cloud system.
+
+
+## Details
+### You will learn
+ - How to configure your CAP application for eventing
+ - How to locally test your changes
+ - How to deploy your CAP application
+
+
+To start with this tutorial use the result in the [`ext-service-s4hc-use`](https://github.com/SAP-samples/cloud-cap-risk-management/tree/ext-service-s4hc-use) branch.
+
```

Comment:
You can remove this line as it is creating lines in the rendered web page.

Reply:
As this comes automatically using a script converting our GitHub content to Atom editor content, we can not remove it. The empty lines are also in other tutorials, like here https://developers.sap.com/tutorials/btp-app-prepare-btp.html.

Reply:
Have you seen what this looks like rendered on the Tutorial Navigator.  It does not look good.  I do recommend removing the extra lines.  Why can you not remove it?  What is preventing you from doing so?

Reply:
removed the lines for now